### PR TITLE
feat: addi18n, add showcase page

### PR DESCRIPTION
**Pull request by Selflocking**
Fix: #26
Fix: #27
Fix: #16
Fix: #14

Because this commit involves i18n, you can preview on http://casbin.yunshu.site.
I translate some items in advance for preview : http://casbin.yunshu.site/zh  http://casbin.yunshu.site/zh/docs/get-started
add users page: http://casbin.yunshu.site/users/

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -67,12 +67,21 @@ module.exports = {
...
+          dropdownItemsAfter: [
+            {
+              type: 'html',
+              value: '<hr style="margin: 0.3rem 0;">',
+            },
+            {
+              href: 'https://crowdin.com/project/casbin-website',
+              label: 'Help Us Translate',
+            },
+          ],
```
Comment (Selflocking) [inline docusaurus.config.js:81]: image

File: docusaurus.config.js [hunk 2]
```diff
@@ -152,11 +161,16 @@ module.exports = {
...
+        '<a href="https://casdoor.org/">💖 Looking for an open-source identity and access management solution like Okta, Auth0, Keycloak ? Learn more about: Casdoor</a>',
       backgroundColor: '#fafbfc',
       textColor: '#091E42', // 默认为 `#000`。
       isCloseable: false, // 默认为 `true`。
     },
+    docs:{
+      sidebar: {
+        hideable: true,
+      },
+    }
```
Comment (Selflocking) [inline docusaurus.config.js:173]: image

File: docusaurus.config.js [hunk 3]
```diff
@@ -171,6 +185,7 @@ module.exports = {
           // Please change this to your repo.
           editUrl:
             'https://github.com/casbin/casbin-website-v2/edit/master/',
+          sidebarCollapsed: false,
```
Comment (Selflocking) [inline docusaurus.config.js:188]: Expand the left-bar menu by default

File: netlify.toml [hunk 4]
```diff
@@ -2,4 +2,4 @@
     ignore = "/bin/false"
     base = "./"
     publish = "build/"
-    command = "yarn build"
\ No newline at end of file
+    command = "yarn build --locale en"
```
Comment (Selflocking) [inline netlify.toml:5]: refer: https://docusaurus.io/docs/i18n/crowdin#automate-with-ci
image

**Comment by hsluoyz** [comment]
@Selflocking well done!

**Pull request merged**